src/ts_catalog/compression_settings.c

```diff
@@ -177,37 +224,66 @@ ts_compression_settings_delete(Oid relid)
 	return count > 0;
 }
 
-TSDLLEXPORT void
-ts_compression_settings_rename_column_hypertable(Hypertable *ht, char *old, char *new)
+/*
+ * Delete entries matching the non-compressed relation.
+ */
+TSDLLEXPORT bool
+ts_compression_settings_delete(Oid relid)
 {
-	ts_compression_settings_rename_column(ht->main_table_relid, old, new);
-	if (ht->fd.compressed_hypertable_id)
-	{
-		ListCell *lc;
-		List *chunks = ts_chunk_get_by_hypertable_id(ht->fd.compressed_hypertable_id);
-		foreach (lc, chunks)
-		{
-			Chunk *chunk = lfirst(lc);
-			ts_compression_settings_rename_column(chunk->table_id, old, new);
-		}
-	}
+	return compression_settings_delete(relid, false);
```

Comment:
This function seems redundant. It is just as easy to call `compression_settings_delete` directly.

Reply:
Answered above.